containers.go

```diff
@@ -868,3 +870,104 @@ func FindDockerImagesByTagPrefix(imageName string) ([]docker.APIImages, error) {
 
 	return matchingImages, nil
 }
+
+// SquashImage takes a docker image with multiple layers and squashes it into
+// a single layer.  SquashImage takes advantage of the fact that docker
+// squashes layers into a single later when exported from a container
+// Note:  It can take minutes to export the container.  Please consider this
+// when setting context.Context.
+func SquashImage(ctx context.Context, repo, tag string) error {
+	client := DockerClient()
```

Comment:
One last thing for consistency with the PR I'm about to send: please also take in the Docker client as a parameter.

```suggestion
func SquashImage(ctx context.Context, client *docker.Client, repo, tag string) error {
```